Update to Django 4.0

Currently RestFramework does not support Django 4.0 deprecation of `pytz`:

```
/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/rest_framework/fields.py:32: in <module>
    from pytz.exceptions import InvalidTimeError
E   ModuleNotFoundError: No module named 'pytz'
```
Update to Django 4.0 when Restframework is released.

--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -1,3 +1,3 @@
-Django>=2.2,<4.0
+Django>=2.2,<=4.0
 djangorestframework>=3.9
 orjson>=2.4

--- a/setup.py
+++ b/setup.py
@@ -28,7 +28,7 @@ setup(
     packages=["drf_orjson_renderer"],
     license="MIT",
     install_requires=[
-        "django>=3.2,<4.0",
+        "django>=3.2,<=4.0",
         "djangorestframework",
         "orjson>=3.3.0",
     ],

